tests/model_selection/splitters/maxmin_split.py

```diff
@@ -0,0 +1,184 @@
+from typing import Union
+
+import pytest
+from rdkit import Chem
+from rdkit.Chem import Mol
+
+from skfp.model_selection.splitters.maxmin_split import (
+    maxmin_train_test_split,
+    maxmin_train_valid_test_split,
+)
+
+
+@pytest.fixture
+def all_molecules() -> list[str]:
+    all_smiles: list[str] = [
```

Comment:
No need for additional typing here, just `return [...]`

Reply:
done